content/python/concepts/sets/terms/union/union.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.union()'
+Description: 'Returns a new set that combines objects from all sets involved, removing any duplicates.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.union()` method returns a new set that combines objects from all [sets](https://www.codecademy.com/resources/docs/python/sets) involved, removing any duplicates.
+
+## Syntax
+
+```pseudo
+set.union(set1, set2, .., setN)
+
+setA | setB | setC
+```
+
+## Example
+
+In the example below, two sets of `liked_artists` are created using two alternate syntaxes. Next, they are combined and stored in a new set, `all_likes`, using the `.union()` method:
+
+```py
+liked_artists_1 = {'Grimes', 'Billie Eilish'}
+liked_artists_2 = set({'Daft Punk', 'Iggy Azalea'})
```

Comment:
It's strange that this second line is using the set() function in conjunction with the default {} set syntax as opposed to applying set() to an alternative collection, such as a tuple or list.

Reply:
That's fair

Reply:
That's fair